triton-vm/src/table/table_column.rs

```diff
@@ -201,39 +187,37 @@ impl std::fmt::Display for ExtProcessorTableColumn {
     }
 }
 
-impl From<ExtProcessorTableColumn> for usize {
-    fn from(c: ExtProcessorTableColumn) -> Self {
-        use ExtProcessorTableColumn::*;
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ProcessorTableColumn {
+    Base(BaseProcessorTableColumn),
+    Ext(ExtProcessorTableColumn),
+}
```

Comment:
This combined enum seems entirely unnecessary.

We only appear to be using the `*TableColumn` types for
- conversion to `usize` when indexing rows
- conversion via `Display` in the stepping debugger

There appears to be no context where we interchangeably refer to either base columns and extension columns. That could have been the case for constraint polynomials that use the extension columns and the base columns, but in that case, the `usize::from(...)` is sufficient on each enum separately.